client/state/selectors/get-sites-for-upcoming-transactions.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { map, uniq } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { getSite } from 'state/sites/selectors';
+import { getUpcomingBillingTransactions } from 'state/selectors';
+
+const getSitesForUpcomingTransactions = createSelector(
+	( state ) => {
+		const siteIds = uniq( map( getUpcomingBillingTransactions( state ), 'blog_id' ) )
+			.map( Number );
+		return siteIds.reduce( ( sites, siteId ) => ( {
+			...sites,
+			[ siteId ]: getSite( state, siteId ),
```

Comment:
:+1: I think there's no reason to store them if they'll be null. Updated in b1554457b95bf72fd8ae63681ef0e555276b7380.